src/main/java/org/apache/maven/plugins/changes/ChangesReport.java

```diff
@@ -218,9 +212,10 @@ public void executeReport(Locale locale) throws MavenReportException {
             }
             final String relativePath = absolutePath.substring(basePath.length());
 
+            ReleaseUtils releaseUtils = new ReleaseUtils(getLog());
```

Comment:
This still calls the instance method in the constructor. This might work, but it's risky and really hard to debug when it doesn't. Could ReleaseUtils set up its own log in the usual way instead?

Reply:
done